protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -301,4 +297,27 @@ public function checkGroupAccess($group)
             throw new HttpException(403);
         }
     }
+
+    /**
+     * Get a restriction message if the Group cannot be deleted
+     *
+     * @param Group $group
+     * @return string|bool Error message if the Group cannot be deleted, false - if no restrictions to delete the Group
+     */
+    protected function getGroupDeleteRestriction($group)
```

Comment:
We should make this private, I think there is no need for a protected visibility. Private functions can be refactored/deleted without introducing breaking changes in the future. If we need it as protected we can change it in the future.

Reply:
Ok, commit https://github.com/humhub/humhub/pull/4879/commits/eac110a6822adbfabbf0ff8e1dc85dd9d70abe3c